Source/View Controllers/TCCProfileViewController.swift

```diff
@@ -351,17 +377,31 @@ extension TCCProfileViewController : NSTableViewDataSource {
         
         guard let url = urls?.first else { return false  }
         
-        guard let newExecutable = model.loadExecutable(url: url) else { return false }
-        
-        if tableView == executablesTable {
-            guard executablesAC.canInsert else { return false }
-            if shouldExecutableBeAdded(newExecutable) {
-                executablesAC.insert(newExecutable, atArrangedObjectIndex: row)
+        guard let window = self.view.window else { return false }
+        var canAdd = true
+        model.loadExecutable(url: url) {
+            [weak self] result in
+            switch result{
```

Comment:
some style related comments:
have a space before the opening brace here.

Line 388 the brace should be on the same line as 387. 

I added a card to put SwiftLint into the project so we can have an enforced style guile for the future:
https://github.com/jamf/PPPC-Utility/issues/55
So not a big deal right now, as that card will clean up any inconsistencies like this.